lgtm.py

```diff
@@ -104,14 +108,15 @@ def force_rebuild_project(self, simple_project: 'SimpleProject'):
         except LGTMRequestException:
             print('Failed rebuilding project. This may be because it is already being built. `%s`' % simple_project)
 
-    def follow_repository(self, repository_url: str):
+    def follow_repository(self, repository_url: str) -> dict:
```

Comment:
I checked and can't find any case where returning data in the function breaks the code.